### ISSUE TITLE
Azure Web App error on start

### ISSUE THREAD

**Issue by Skrypt** [OrchardCMS/OrchardCore#3834]
This error gets added to log every time I start the web app.

```
2019-06-20 03:03:01.0408||||Microsoft.AspNetCore.Server.IIS.Core.IISHttpServer|ERROR|Unexpected exception in IISHttpServer.HandleRequest. System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.AspNetCore.Server.IIS.Core.IISHttpContext.GetOriginalPath()
   at Microsoft.AspNetCore.Server.IIS.Core.IISHttpContext.InitializeContext()
   at Microsoft.AspNetCore.Server.IIS.Core.IISHttpContextOfT`1.ProcessRequestAsync()
   at Microsoft.AspNetCore.Server.IIS.Core.IISHttpServer.HandleRequest(IISHttpContext context)    at Microsoft.AspNetCore.Server.IIS.Core.IISHttpContext.GetOriginalPath()
   at Microsoft.AspNetCore.Server.IIS.Core.IISHttpContext.InitializeContext()
   at Microsoft.AspNetCore.Server.IIS.Core.IISHttpContextOfT`1.ProcessRequestAsync()
   at Microsoft.AspNetCore.Server.IIS.Core.IISHttpServer.HandleRequest(IISHttpContext context)
```

**Comment by deanmarcussen** [comment]
What framework version are you running?

I get this on one deployment during startup that is on 4.7

```
2019-06-20 01:41:33.8968|None|80003660-0003-f000-b63f-84710c7967bb||OrchardCore.Environment.Shell.ShellHost|INFO|Start creation of shells 
2019-06-20 01:41:33.8968||||Microsoft.AspNetCore.Hosting.Internal.WebHost|FATAL|Hosting startup assembly exception System.InvalidOperationException: Startup assembly Microsoft.AspNetCore.AzureAppServices.HostingStartup failed to execute. See the inner exception for more details. ---> System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.AspNetCore.AzureAppServices.HostingStartup, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.
   at System.Reflection.RuntimeAssembly.nLoad(AssemblyName fileName, String codeBase, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, IntPtr ptrLoadContextBinder)
   at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, IntPtr ptrLoadContextBinder)
   at System.Reflection.Assembly.Load(AssemblyName assemblyRef)
   at Microsoft.AspNetCore.Hosting.WebHostBuilder.BuildCommonServices(AggregateException& hostingStartupErrors)
```

but I don't get it on a more recent webapp that is (probably) running core 2.2 properly

If you change your appsettings to log some of the microsoft debug / info settings you might get a better stack trace of what's trying to load a missing file?

**Comment by Skrypt** [comment]
It's running on .Net Core 2.2 without any refs folder.

**Comment by jtkech** [comment]
This was an known [issue on aspnetcore](https://github.com/aspnet/AspNetCore/issues/6075), need to leave soon but i will read it this night.

**Comment by sebastienros** [comment]
It's on aspnet's side. Will be fixed once we are on 3.0, and it's just log.
